app/institutions/dashboard/-components/institutional-dashboard-wrapper/template.hbs

```diff
@@ -0,0 +1,84 @@
+<OsfLayout @backgroundClass='container' as |layout|>
+    <layout.heading
+        local-class='heading-wrapper'
+    >
+        <div local-class='banner'>
+            <img src='{{@institution.bannerUrl}}' alt='{{@institution.name}}'>
+            <div>
+                {{t 'institutions.dashboard.last_update'}}
+            </div>
+        </div>
+        <ul
+            data-analytics-scope='Dashboard tabs'
+            local-class='tab-list'
+        >
+            <li
+                local-class='tab'
+            >
+                <OsfLink
+                    data-analytics-name='Summary'
+                    data-test-page-tab='summary'
+                    @route='institutions.dashboard.index'
+                    @model={{@institution}}
```

Comment:
this doesn't seem to line up with [the expected model](https://github.com/CenterForOpenScience/ember-osf-web/blob/f6bd878b132569262d2bee6f28da5a8440eed2cd/app/institutions/dashboard/index/template.hbs#L5), does the link work as expected?

(tho honestly i'd be a fan of stopping with the `{taskInstance: ...}` model pattern and using actual model instances instead -- would simplify this and other logic)